cmd/make-bm-worker/main.go

```diff
@@ -9,6 +9,11 @@ import (
 	"text/template"
 )
 
+const (
+	instanceImageSource      = "http://172.22.0.1/images/rhcos-ootpa-latest.qcow2"
+	instanceImageChecksumURL = instanceImageSource + ".md5sum"
```

Comment:
The image related information is specific to the dev-scripts setup and using this code to deploy RHCOS nodes. We don't want to put that in this upstream repo.

Reply:
Updated